src/main/java/mate/academy/spring/model/Role.java

```diff
@@ -0,0 +1,65 @@
+package mate.academy.spring.model;
+
+import java.util.Objects;
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import org.springframework.security.core.GrantedAuthority;
+
+@Entity
+public class Role implements GrantedAuthority {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @Enumerated(value = EnumType.STRING)
```

Comment:
Consider adding @Column(name = "role_name") annotation to make column name explicit and avoid possible issues with automatic naming conversions.